src/ariel-os-rt/src/lib.rs

```diff
@@ -63,10 +63,13 @@ mod isr_stack {
         CONFIG_ISR_STACKSIZE
     };
 
-    // SAFETY: a linker script inserts this section in RAM.
-    #[unsafe(link_section = ".isr_stack")]
-    #[used(linker)]
-    static ISR_STACK: [u8; ISR_STACKSIZE] = [0u8; ISR_STACKSIZE];
+    core::arch::global_asm!(
+    r#"
+        .section .isr_stack, "wa"
+        .skip {size}
+    "#,
```

Comment:
Desired indentation:

```suggestion
        r#"
        .section .isr_stack, "wa"
        .skip {size}
        "#,
```

Reply:
ah ok, will do

Reply:
done